math/mathcore/inc/Fit/Fitter.h

```diff
@@ -462,6 +462,21 @@ class Fitter {
    */
    bool ApplyWeightCorrection(const ROOT::Math::IMultiGenFunction & loglw2, bool minimizeW2L=false);
 
+   /// Set number of fit points when using an external FCN function
+   /// This function can be called after Fit to set the correct number of Ndf in FitResult
+   void SetNumberOfFitPoints(unsigned int npoints) {
```

Comment:
Does it even make sense to call this when you have no `fExtObjFunction`? If not, it could throw an exception in this case

Reply:
No, because there could be case where a FCN is created internally or provided as an external function, but still the number of fit point and fit type is not defined

Reply:
Okay, thanks for clarifying